packages/components/search/src/search.js

```diff
@@ -85,14 +145,36 @@ export class TSSearch extends TSElement {
 			case 'focused':
 				if (this.focused) {
 					// set focus only when component is defined.
-					window.customElements.whenDefined(ELEMENT_TAG).then(() => this.shadowRoot.getElementById('input').focus());
+					window.customElements.whenDefined(ELEMENT_TAG).then(() => {
+						this.shadowRoot.getElementById('input').focus();
+						if (this.showSuggestions && this.focused) {
+							this.opened = true;
+						} else {
+							this.opened = null;
+						}
```

Comment:
Not boolean? Does it need a third state (the null)? If boolean, it could be:

this.opened = this.showSuggestions && this.focused